src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -118,4 +119,11 @@
      */
     void clearTitle();
 
+    /**
+     * Gets the appropriate {@link PlayerNetworkHandler} linking this Player
+     * to a client.
+     *
+     * @return The connection
+     */
+    PlayerNetworkHandler getNetwork();
```

Comment:
Personally I think this name is a little bit misleading. The "network" is the component handling connections, processing packets etc.. In terms of this API, `PlayerNetworkHandler` doesn't represent the handler of the connection, but rather the connection itself. Because of this, in my opinion the name `PlayerConnection` or similar as used before is far more appropriate than this one.